src/guiguts/application.py

```diff
@@ -805,7 +805,7 @@ def init_view_menu(self) -> None:
     def init_help_menu(self) -> None:
         """Create the Help menu."""
         menu_help = Menu(menubar(), "~Help")
-        menu_help.add_button("Guiguts ~Manual", self.show_help_manual)
+        menu_help.add_button("Guiguts ~Manual", self.show_help_manual, "F1")
```

Comment:
I think this being in the menu means it is fired even when the main window isn't focused. See my main comment about multiple pages being opened.

Perhaps instead of binding here in the menu, remove this menu-based binding and put an F1 bind on the main window.

Reply:
Decided I'd prefer the "F1" accelerator to continue to appear in the menu (if only to remind users which key it is to get help), so I've added an arg to `add_button` to force binding only to main window (which it already did for inbuilt virtual events anyway)